Framework/Algorithms/src/SortXAxis2.cpp

```diff
@@ -0,0 +1,257 @@
+#include "MantidAlgorithms/SortXAxis2.h"
+#include <iostream>
```

Comment:
`<iostream>` is heavy weight and shouldn't be required for normal operation. Can we remove it?